src/config.rs

```diff
@@ -286,7 +291,7 @@ impl Config {
                         Value::String(s) => {
                             let extension = Path::new(&s).extension();
                             if extension.is_some() && extension.unwrap() == "jar" {
-                                config.apktool_file = s.clone();
+                                config.apktool_file = PathBuf::from(s.clone());
```

Comment:
The borrow of `Path::new(&s)` requires a clone of `s`. However, one could create the PathBuf upfront and use `extension()` directly on it. Probably applicable to the other cases as well.